plugins/dynamic-home-page/catalog-info.yaml

```diff
@@ -0,0 +1,61 @@
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: janus-idp-dynamic-home-page
+  title: Dynamic Home Page plugin
+  description: Home Page plugin based on the upstream home [page] plugin that can be extended via (dynamic) plugins.
+  annotations:
+    backstage.io/source-location: url:https://github.com/janus-idp/backstage-showcase/tree/main/plugins/dynamic-home-page
+    #backstage.io/source-location: file:../../plugins/dynamic-home-page/REQUIRED_TO_BUILD_THE_TECHDOCS_LOCALLY
+    backstage.io/view-url: https://github.com/janus-idp/backstage-showcase/blob/main/plugins/dynamic-home-page/catalog-info.yaml
+    backstage.io/edit-url: https://github.com/janus-idp/backstage-showcase/edit/main/plugins/dynamic-home-page/catalog-info.yaml
+    backstage.io/techdocs-ref: dir:.
+    github.com/project-slug: janus-idp/backstage-showcase
+    github.com/team-slug: janus-idp/rhtap
+    sonarqube.org/project-key: janus-idp_backstage-showcase
+  tags:
+    - homepage
+  links:
+    - url: https://github.com/janus-idp/backstage-showcase/tree/main/plugins/dynamic-home-page
+      title: GitHub Source
+      icon: source
+      type: source
+spec:
+  type: backstage-plugin
+  lifecycle: production
+  owner: rhdh-team
+  system: rhdh
+  subcomponentOf: janus-idp-backstage-plugins
+---
+# https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: janus-idp-dynamic-home-page-frontend
+  title: '@janus-idp/backstage-plugin-dynamic-home-page'
+  description: Home Page plugin based on the upstream home [page] plugin that can be extended via (dynamic) plugins.
+  annotations:
```

Comment:
how is this different from the above component?

Reply:
One is backstage-plugin (something we should show users)

```
  name: janus-idp-dynamic-home-page
  title: Dynamic Home Page plugin
```

The other one is the frontend-plugin:

```
  name: janus-idp-dynamic-home-page-frontend
  title: '@janus-idp/backstage-plugin-dynamic-home-page'
```